src/amo/components/GetFirefoxButton/index.js

```diff
@@ -37,45 +47,64 @@ export type Props = {|
   addon?: AddonType,
   buttonType: GetFirefoxButtonTypeType,
   className?: string,
+  useNewVersion?: boolean,
```

Comment:
The current state is probably okay but some of the mocks have an icon in the yellow thing: https://www.figma.com/file/7FnxRoHYpyEfMOIfob423s/Non-Firefox-Download?node-id=323%3A61

Reply:
> The current state is probably okay but some of the mocks have an icon in the yellow thing: [figma.com/file/7FnxRoHYpyEfMOIfob423s/Non-Firefox-Download?node-id=323%3A61](https://www.figma.com/file/7FnxRoHYpyEfMOIfob423s/Non-Firefox-Download?node-id=323%3A61)

Those are quite old. The newer ones don't have the icon, and screenshots of the version from this PR have been approved at https://github.com/mozilla/addons-frontend/issues/10252#issuecomment-807440995

Reply:
@bobsilverberg is it expected to have a different yellow?

Reply:
Based on the discussion in Slack I have updated it to use the new yellow and the icon.